app/assets/javascripts/details/state/form_change_state.js

```diff
@@ -14,6 +14,7 @@ FormChangeState = (function(){
 
   FormChangeState.prototype.initDirrty = function(){
     this.form = this.el.dirrty();
+    $('[data-is-dirrty]').attr('data-is-dirrty', 'false');
```

Comment:
When forms were saving, the fields weren't reverting to "not dirty", until clicked.